Command/RunCommand.php

```diff
@@ -428,9 +429,25 @@ private function getCommandProcessBuilder()
             $pb->add('exec');
         }
 
+        // Localize the `console` command
```

Comment:
This looks much more complicated as the user can use either ``php console`` or ``console`` so we would have to parse the input.